### PR TITLE
Add robots.txt patch v1

### PR DESCRIPTION
**Pull request by prahaladbelavadi**
Importing from https://github.com/k9mail/k9mail.github.io/pull/96 

Objective: to facilitate atomic commits; Was clashing with another pull request on license addition

Review needed from @BO41

### CODE REVIEW

File: robots.txt [hunk 1]
```diff
@@ -1,3 +1,3 @@
 User-agent: *
-Disallow: /
-Sitemap: http://k9mail,github.io/sitemap.xml
+Disallow: 
+Sitemap: http://k9mail.github.io/sitemap.xml
```
[the remark this answers is not in the document]
Comment (prahaladbelavadi) [reply robots.txt]: Opened a PR at #99 that adds a site.xml file

File: robots.txt [hunk 2]
```diff
@@ -0,0 +1,3 @@
+User-agent: *
+Disallow: 
+Sitemap: http://k9mail.github.io/sitemap.xml
```
Comment (BO41) [inline robots.txt]:
```suggestion
Sitemap: https://k9mail.github.io/sitemap.xml
```
to prevent a redirect

Comment (prahaladbelavadi) [reply robots.txt]: Makes sense. Making the changes now

Comment (prahaladbelavadi) [reply robots.txt]: Changed Http to https

**Pull request closed without merging**